server/src/main/java/io/crate/role/GrantedRolesChange.java

```diff
@@ -29,19 +29,19 @@
 import org.elasticsearch.common.io.stream.StreamOutput;
 import org.elasticsearch.common.io.stream.Writeable;
 
-public class RolePrivilegeToApply implements Writeable {
+public class GrantedRolesChange implements Writeable {
```

Comment:
Would `GrantsChange` or `RolesChange` not also communicate the same information?

Reply:
On second thought, `RolesChange` would be ambigous and could refer to changing a dedicated roles.

Reply:
`GrantsChange`, can also refer to simple `Privilege`, so I prefer the `GrantedRolesChange` to be honest. `RolesChange` is closer, but again (just from the name itself) could refer to changing properties of the Roles.